src/metal.cs

```diff
@@ -4397,25 +4639,32 @@ interface MTLRasterizationRateLayerDescriptor : NSCopying {
 		[Internal]
 		[Export ("initWithSampleCount:horizontal:vertical:")]
 		IntPtr Constructor (MTLSize sampleCount, IntPtr horizontal, IntPtr vertical);
-
-/* Selectors reported as not working by introspection: https://github.com/xamarin/maccore/issues/1976
```

Comment:
This code should be enabled too:

https://github.com/xamarin/xamarin-macios/blob/9286bfb79309811b96ece9a283f1d13d5f7c52e3/src/Metal/MTLRasterizationRateLayerDescriptor.cs#L15-L35